CalendarApp/CalendarGoogleService/GoogleService.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Google.Apis.Calendar.v3;
+using Google.Apis.Calendar.v3.Data;
+using log4net;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalendarGoogleService
+{
+    public class GoogleService
```

Comment:
Rename it to smth more specific like MeetingBooking service, or CalendarManagementService etc